profiles/logical/holo/hydra/master/default.nix

```
@@ -109,6 +115,17 @@ in
     };
   };
 
+  services.restic.backups = {
+    remotebackup = {
+      user = "root";
+      paths = [ "/var/backup/postgresql/" ];
+      s3CredentialsFile = "/var/lib/hydra/pgbackup/keys/hydra-master-pgbackup/secret";
+      repository = "s3:https://${wasabiEndpoint}/${wasabiPgBucket}";
```

As far as I can tell, endpoint we end up using is different because our backup bucket is in `us-east-2`, while `wasabiEndpoint` corresponds to `us-east-1`: https://wasabi-support.zendesk.com/hc/en-us/articles/360015106031-What-are-the-service-URLs-for-Wasabi-s-different-regions-

Also let's just inline the bucket name since endpoint is different anyway.

```suggestion
      repository = "s3:https://s3.us-east-2.wasabisys.com/hydra-master-restic";
```

---

I actually tested this with the previous endpoint and confirmed that it worked and made it to the correct endpoint destination.

However I will commit this, and the rest of the suggestions, rebase from this, and retest on local just to be sure that it works.